pcawg_metadata_parser/pc_report-sanger_summary_counts.py

```diff
@@ -98,13 +100,20 @@ def compute_site_report_new(metadata_dir, report_dir, today_donors):
         'Total': len(today_donors[0].union(today_donors[1]).difference(site_assigned_donors))
     }
 
+    unassigned_uncalled_donors = today_donors[0].difference(site_assigned_donors)
+
     # today's counts
     with open(report_dir + '/summary_compute_site_counts.json', 'w') as o: o.write(json.dumps(site_summary))
 
-    # get all previous days counts
-    [dates, metadata_dirs] = get_metadata_dirs(metadata_dir, '2015-03-07')
+    with open(report_dir + '/unassigned_uncalled_donors.txt', 'w') as o:
+        o.write('# Unassigned and uncalled donors\n')
+        o.write('# dcc_project_code' + '\t' + 'submitter_donor_id' + '\n')
+        o.write('\n'.join(unassigned_uncalled_donors) + '\n') 
 
     """
+    # get all previous days counts
+    [dates, metadata_dirs] = get_metadata_dirs(metadata_dir, '2015-04-01')
```

Comment:
Hi Junjun,

This part was commented out. I changed the date just for running this code on my computer locally, because I have no history data of March.

-Linda



Linda Xiang
Bioinformatician

Ontario Institute for Cancer Research
MaRS Centre
661 University Avenue
Suite 510
Toronto, Ontario
Canada M5G 0A3

Email: Linda.Xiang@oicr.on.ca<mailto:Linda.Xiang@oicr.on.ca>
Twitter: @OICR_news

www.oicr.on.ca<http://www.oicr.on.ca>

This message and any attachments may contain confidential and/or privileged information for the sole use of the intended recipient. Any review or distribution by anyone other than the person for whom it was originally intended is strictly prohibited. If you have received this message in error, please contact the sender and delete all copies. Opinions, conclusions or other information contained in this message may not be that of the organization.



On Jun 11, 2015, at 3:28 PM, Junjun Zhang <notifications@github.com<mailto:notifications@github.com>> wrote:


In pcawg_metadata_parser/pc_report-sanger_summary_counts.py<https://github.com/ICGC-TCGA-PanCancer/pancancer-sandbox/pull/36#discussion_r32258359>:

>
>      """
> +    # get all previous days counts
> +    [dates, metadata_dirs] = get_metadata_dirs(metadata_dir, '2015-04-01')


Any reason the date for start tracking Sanger variant calling needs to be updated?

—
Reply to this email directly or view it on GitHub<https://github.com/ICGC-TCGA-PanCancer/pancancer-sandbox/pull/36/files#r32258359>.